src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -1348,6 +1361,27 @@ private bool BelongsToGroup()
             return WorkspaceViewModel.Model.Annotations.ContainsModel(this.annotationModel);
         }
 
+        internal void ToggleIsVisibleGroup(object parameters)
+        {
+            DynamoSelection.Instance.ClearSelection();
```

Comment:
why is it necessary to clear the selection?

Reply:
Just to clear any other groups/notes/nodes that may be present in the selection, as we only want to target the current group and its contents.